### PR TITLE
Mneuss/negation 06 third participation questions feature

### PR DESCRIPTION
**Pull request by m-neuss**

### CODE REVIEW

File: src/color-toj-negation-06.js [hunk 1]
```diff
@@ -471,13 +473,134 @@ Falls Sie für üblich eine Brille tragen, setzen Sie diese bitte für das Exper
...
+    conditional_function: () => globalProps.isThirdParticipation === true,
+    timeline: [
+      {
+        type: "survey-text",
+        questions: () => {
+          if (globalProps.instructionLanguage === "en") {
+            return [
+              { 
+                name: "Do you have any guesses as to what we might be investigating and what might come out of it?",
+                prompt: "<p>Do you have any guesses as to what we might be investigating and what might come out of it?</p>",
```
Comment (chisburger) [inline src/color-toj-negation-06.js]:
```suggestion
                prompt: "<p>What do you reckon we are investigating? What do you think might be the result of the study?</p>", 
```

File: src/color-toj-negation-06.js [hunk 2]
```diff
@@ -471,13 +473,134 @@ Falls Sie für üblich eine Brille tragen, setzen Sie diese bitte für das Exper
...
+  const thirdParticipationQuestions = {
+    conditional_function: () => globalProps.isThirdParticipation === true,
+    timeline: [
+      {
+        type: "survey-text",
+        questions: () => {
+          if (globalProps.instructionLanguage === "en") {
+            return [
+              { 
+                name: "Do you have any guesses as to what we might be investigating and what might come out of it?",
```
Comment (chisburger) [inline src/color-toj-negation-06.js]:
```suggestion
                name: "What do you reckon we are investigating? What do you think might be the result of the study?",
```

File: src/color-toj-negation-06.js [hunk 3]
```diff
@@ -471,13 +473,134 @@ Falls Sie für üblich eine Brille tragen, setzen Sie diese bitte für das Exper
...
+          };
+        },
+      },
+      {
+        type: "survey-text",
+        questions: () => {
+          if (globalProps.instructionLanguage === "en") {
+            return [
+              { 
+                name: "Estimate: How often were negations (not red or not green) mentioned in direct successive stages?",
```
Comment (chisburger) [inline src/color-toj-negation-06.js]:
```suggestion
                name: "Please estimate: How often were negations (not red or not green) said in successive trials?",
```

File: src/color-toj-negation-06.js [hunk 4]
```diff
@@ -471,13 +473,134 @@ Falls Sie für üblich eine Brille tragen, setzen Sie diese bitte für das Exper
...
+                prompt: "<p>Estimate: How often were negations (\"not red\" or \"not green\") mentioned in direct successive stages?</p>",
```
Comment (chisburger) [inline src/color-toj-negation-06.js]:
```suggestion
                prompt: "<p>Please estimate: How often were negations (\"not red\" or \"not green\") said in successive trials?</p>", 
```

File: src/color-toj-negation-06.js [hunk 5]
```diff
@@ -471,13 +473,134 @@ Falls Sie für üblich eine Brille tragen, setzen Sie diese bitte für das Exper
...
+          };    
+        },
+      },
+      {
+        type: "survey-text",
+        questions: () => {
+          if (globalProps.instructionLanguage === "en") {
+            return [
+              { 
+                name: "Estimate: How often were sentences without negation (now red or now green) mentioned in direct successive stages?",
```
Comment (chisburger) [inline src/color-toj-negation-06.js]:
```suggestion
                name: "Please estimate: How often were sentences without negation (now red or now green) said in successive trials?",
```

File: src/color-toj-negation-06.js [hunk 6]
```diff
@@ -471,13 +473,134 @@ Falls Sie für üblich eine Brille tragen, setzen Sie diese bitte für das Exper
...
+                prompt: "<p>Estimate: How often were sentences without negation (\"now red\" or \"now green\") mentioned in direct successive stages?</p>",
```
Comment (chisburger) [inline src/color-toj-negation-06.js]:
```suggestion
                prompt: "<p>Please estimate: How often were sentences without negation (\"now red\" or \"now green\") said in successive trials?</p>", 
```

File: src/util/introduction.js [hunk 7]
```diff
@@ -93,15 +97,23 @@ export function addIntroduction(timeline, options) {
...
-            prompt:
-              "<p>Please enter your participant code (the one you got the first time you participated in this experiment).</p>",
-            required: true,
-          },
-        ],
+        questions: () => {
+          if (globalProps.instructionLanguage === "en") {
+            return [{ 
+              prompt: 
+                "<p>Please enter your participant code (the one you got the first time you participated in this experiment).</p>",
```
Comment (chisburger) [inline src/util/introduction.js]:
```suggestion
                "<p>Please enter your participant code (that you got the first time you participated in this experiment).</p>",
```

**Review by chisburger — CHANGES_REQUESTED** [review]

**Comment by caiusno1** [comment]
Works in all testcases which I have applied. 
Therefore I approve the changes

**Review by caiusno1 — APPROVED** [review]

**Review by chisburger — APPROVED** [review]

**Pull request merged**